__tests__/api.test.js

```diff
@@ -96,22 +97,25 @@ describe("GET /api/articles", () => {
 });
 
 describe("GET /api/articles/:article_id/comments", () => {
-  test("GET:200 gets comnments of a specific article", () => {
+  test("GET:200 gets comments of a specific article", () => {
     return request(app)
-      .get("/api/articles/2/comments")
+      .get("/api/articles/1/comments")
       .expect(200)
       .then((res) => {
-        articles = [res.body.article];
-        articles.forEach((article) => {
-          expect(article).toHaveProperty("comment_id");
-          expect(article).toHaveProperty("votes");
-          expect(article).toHaveProperty("created_at");
-          expect(article).toHaveProperty("author");
-          expect(article).toHaveProperty("body");
-          expect(article).toHaveProperty("article_id");
+        const comments = res.body.article;
+        console.log(typeof comments);
+        comments.forEach((comment) => {
```

Comment:
We should be checking the length of the comments coming back before running a for each! Otherwise we leave ourselves open to tests passing when they shouldn't! 

This is because if an empty array comes back then the forEach will not invoke it's own callback function which contains our assertions. If a test has no assertions, then jest will pass it!